core/domain/collection_domain.py

```diff
@@ -1064,8 +1060,14 @@ def delete_node(self, exploration_id):
         del self.nodes[node_index]
 
     def add_skill(self, skill_name):
-        """Adds the new skill domain object with the specified name."""
+        """Adds the new skill domain object with the specified name.
 
+        Args:
+            skill_name: str. The name of the skill.
+
+        Returns
+            Id of the new skill.
```

Comment:
Here and elsewhere: missing type of return value.

Reply:
Updated.